src/languages/cs.js

```diff
@@ -111,15 +124,40 @@ function(hljs) {
       {
         className: 'meta',
         begin: '#', end: '$',
-        keywords: {'meta-keyword': 'if else elif endif define undef warning error line region endregion pragma checksum'}
+        keywords: {
+          'meta-keyword': 'if else elif endif define undef warning error line region endregion pragma checksum'
+        }
       },
       STRING,
       hljs.C_NUMBER_MODE,
       {
-        beginKeywords: 'class interface', end: /[{;=]/,
+        beginKeywords: 'class', end: /[{;=]/,
         illegal: /[^\s:]/,
         contains: [
-          hljs.TITLE_MODE,
+          {
+            // for inherited interface in class declaration
+            className: 'class',
+            begin: 'I[A-Z]\\w*',
+            relevance: 0
+          },
+          {
+            className: 'type',
+            begin: hljs.IDENT_RE,
+            relevance: 0
+          },
```

Comment:
Same thing here, "type" and "class" are reserved for different things, and here it really should be "title". This is just a design decision made in highlight.js and it has to be consistent across similar languages.